src/components/chip/examples/chip-link.tsx

```diff
@@ -0,0 +1,42 @@
+import { Component, h } from '@stencil/core';
+
+/**
+ * Chip as hyperlink
+ * A chip can be used to navigate to a different page, pane of the application
+ * or to an external site. To get the chip to act as a hyperlink, you need to
+ * provide a `Link`.
+ *
+ * This way, the component will automatically become a native HTML link.
+ * This gives the users the possibility of interacting with links
+ * in a natural way, for instance they can open any of the previous
+ * pages in a new browser tab. This can also have other accessibility benefits.
```

Comment:
> to navigate to a different page, pane of the application or to an external site

Nitpick, but this sounds a bit clunky, I think, especially "pane of the application".  I understand that refers to the sidepanel in the case of Lime CRM, but without that reference, it would just seem a bit odd. It's a little unnecessary to describe what _kind_ of navigation can be performed with a link, as well. Like, "we know, it's called _the web_" 😅

> for instance they can open any of the previous pages in a new browser tab

🤨 😅

Overall, I would probably rephrase this whole section a bit. I'd like it more imperative (is that the right word for it?). Something like:

> For accessibility and usability alike, if clicking on a chip should result in any kind of navigation, it is preferable to use a link, rather than a button. That way, the user can choose to, for example, open the link in a new tab.
>
> For this reason, we suggest always providing a `Link` with the URL representing the target state of the navigation, if such a URL exists.

Although less pompously worded than I managed here perhaps… 🙈

Reply:
agree! ⚡